src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeFeature.cs

```diff
@@ -47,14 +47,31 @@ public static partial class RuntimeFeature
         /// </summary>
         public const string NumericIntPtr = nameof(NumericIntPtr);
 
+        /// <summary>
+        /// Indicates that this version of the runtime supports async methods.
+        /// </summary>
+        [RequiresPreviewFeatures]
+        public const string Async = nameof(Async);
```

Comment:
Are we going back to gating the support on `RuntimeFeature`s flag?

https://github.com/dotnet/runtime/issues/114310#issuecomment-2797863342

Reply:
No, sorry, looks like some of this is out of date. I’ll fix it up